ci-operator/step-registry/ipi/conf/azure/multidisk/swap/ipi-conf-azure-multidisk-swap-commands.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+function swap_machineconfig_generate(){
+    local role=$1 config_file=$2
+    cat >> ${config_file} << EOF
+apiVersion: machineconfiguration.openshift.io/v1
+kind: KubeletConfig
+metadata:
+  name: 99-swap-config-${role}
+spec:
+  machineConfigPoolSelector:
+    matchLabels:
+      pools.operator.machineconfiguration.openshift.io/${role}: ""
+  kubeletConfig:
+    failSwapOn: false 
+    memorySwap:
+      swapBehavior: LimitedSwap
+EOF
+}
+
+cat >> "${SHARED_DIR}/99-swap-memory-enabled-swapaccount.yaml" << EOF
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
```

Comment:
This machineconfig would be applied all the nodes in the cluster ?

Reply:
Can we apply the setting to the needed nodes ?

Reply:
I could see that those configurations in the cluster, the change should be worked.
```
$ oc get kubeletconfig
NAME                    AGE
99-swap-config-master   2m23s
99-swap-config-worker   2m23s

$ oc get machineconfig
NAME                           GENERATEDBYCONTROLLER   IGNITIONVERSION   AGE
01-disk-setup-swap-master                              3.2.0             2m21s
01-disk-setup-swap-worker                              3.2.0             2m20s
99-installer-ignition-master                           3.2.0             2m22s
99-master-ssh                                          3.2.0             2m20s
99-worker-ssh                                          3.2.0             2m20s
```